### PR TITLE
Update links to graphql client repository

### PR DESCRIPTION
**Pull request by th1988**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ### (Next)
 * Your contribution here.
+* Update graphql-client github repository links in README - [@th1988]
```
Comment (yuki24) [inline CHANGELOG.md:3]: I'm not sure if this deserves a CHANGELOG entry, as there is no user-facing change in the code.

@dblock @ashkan18 Any thoughts?

**Review by yuki24 — APPROVED** [review]
I had a question about the CHANGELOG entry, but overall 👍

File: CHANGELOG.md (hunk 1 above)
Comment (dblock) [reply CHANGELOG.md:3]: It's fine, but we do need to fix CI even if this is unrealated.

**Review by dblock — CHANGES_REQUESTED** [review]
Please make the CHANGELOG line like the others, with link to PR and your name, thanks!

We do need to fix CI, I know it's not failing because of this change, but I'd prefer we don't merge anything before that. Maybe you can help?

**Comment by yuki24** [comment]
@dblock The CI builds should be fixed by https://github.com/ashkan18/graphlient/pull/113.

**Comment by dblock** [comment]
Rebase this please?

**Comment by th1988** [comment]
Done @dblock 🚀

**Review by dblock — APPROVED** [review]

**Pull request merged**